opennms-webapp/src/main/java/org/opennms/web/controller/SendEventController.java

```diff
@@ -93,8 +96,11 @@ private String buildEventSelect() throws IOException, FileNotFoundException {
             // System.out.println(trimmedUei);
 
             if (!excludeList.contains(trimmedUei)) {
-                sortedMap.put(label, uei);
-                // System.out.println("sortedMap.put('"+label+"', '"+uei+"')");
+                try {
```

Comment:
Don't insert if label is null, log a warning in else case

Reply:
Done